src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -92,14 +92,15 @@ protected function getClassMetadata(string $resourceClass): ClassMetadata
      * Determines whether the given property is enabled.
      *
      * @param string $property
+     * @param string $resourceClass
      *
      * @return bool
      */
-    protected function isPropertyEnabled(string $property): bool
+    protected function isPropertyEnabled(string $property, string $resourceClass): bool
```

Comment:
We cannot change the signature of a `protected` method like this. It will cause a BC break. You can handle this like that: https://github.com/symfony/symfony/pull/19371/files#diff-eda6922ceec0d4a52894dd37d7772dc1R180